services/viva/api/cases/serverless.yml

```diff
@@ -0,0 +1,83 @@
+service: viva-cases-api
+
+plugins:
+  - serverless-bundle
+
+projectDir: ../../../../
+
+custom: ${file(../../../../serverless.common.yml):custom}
+
+package:
+  individually: true
+
+provider:
+  lambdaHashingVersion: 20201221
+  name: aws
+  runtime: nodejs14.x
+  stage: dev
+  region: eu-north-1
+  endpointType: regional
+  eventBridge:
+    useCloudFormation: true
+  tracing:
```

Comment:
I don't think that tracing i necessary to use since we are not using x-ray. And to use x-ray in the lambda function you need to use the x-ray client, which we aren't

Reply:
Future implementation. Will not remove